source/api_documentation/network_integration/call_activity_tracking/index.rst

```diff
@@ -0,0 +1,51 @@
+Call Activity Tracking
+======================
+
+Manage call activity tracking for campaigns
+"""""""""""""""""""""""""""""""""""""""""""
+
+Call activity tracking can be set for a campaign to determine whether or not call activity tracking should be enabled. The call activity will be tracked if the full condition is met.
+
+Set Call Activity Tracking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To conditionally enable call activity tracking, set campaign's owning network `call_activity_access` feature flag to `true`, and set ``"call_activity_tracking"`` to ``true``.
+
+POST
+
+``https://invoca.net/api/@@CAMPAIGN_FEATURES_API_VERSION/<network_id>/advertisers/<advertiser_id_from_network>/advertiser_campaigns/<advertiser_campaign_id_from_network>.json``
+
+Example Request Body
+
+.. code-block:: json
+
+  {
+    "ivr_tree": {
+      "call_activity_tracking": true,
```

Comment:
You can use one of the tracking options here as an example
```
"call_activity_tracking": "NoTracking",
```

Reply:
Fixed, thank you!